roles/installer/templates/postgres.yaml.j2

```diff
@@ -37,6 +37,26 @@ spec:
       imagePullSecrets:
         - name: {{ image_pull_secret }}
 {% endif %}
+      initContainers:
+        - name: database-check
+          image: '{{ _init_container_image }}'
+          resources:
+            requests:
+              memory: "64Mi"
+              cpu: "250m"
+            limits:
+              memory: "256Mi"
```

Comment:
After looking at this a bit more closely, I dont see why we would want hardcoded limits on this container when we dont have them on any of the others. I would just remove these.